src/main/java/org/concordion/internal/command/ExampleCommand.java

```diff
@@ -39,7 +40,11 @@ public void execute(CommandCall node, Evaluator evaluator, ResultRecorder result
         }
 
         try {
-            node.getChildren().processSequentially(evaluator, resultRecorder);
+            if (exampleFilter != null && exampleFilter.shouldSkip(node.getElement())) {
```

Comment:
This is nice - one day we'll use a version of Java with Lambdas and convert the exampleFilter to one of the standard interfaces.